tracee-rules/input.go

```diff
@@ -66,8 +67,118 @@ func setupTraceeSource(traceeFilePath string) (chan types.Event, error) {
 				res <- event
 			}
 		}
-		f.Close()
+		opts.inputFile.Close()
```

Comment:
since you have unified stdin and file, you will now close stdin. I'm not sure whether this is good or not

Reply:
I think that's fine. At this point in the code we're going to be closing down so we won't need to read from this stdin fd anyway. Does feel a little weird though.

Reply:
What about the case when there are multiple input sources? Not sure we can assume we're closing down
Also see my answer in #508

Reply:
yes we can't assume we're closing the program but I guess closing the file won't do any harm